packages/office-ui-fabric-react/src/components/Calendar/Calendar.tsx

```diff
@@ -115,6 +115,7 @@ export class Calendar extends BaseComponent<ICalendarProps, ICalendarState> impl
                   onNavigateDate={ this._onNavigateDate } /> }
 
                 { showGoToToday && <span
+                  role='button'
```

Comment:
done. Apart from this I have converted days and months role to match http://whatsock.com/tsg/Coding%20Arena/ARIA%20Date%20Pickers/ARIA%20Date%20Picker%20%28Basic%29/demo.htm. We don't need a table layout here.